digraph "unitGraph" {
    "r36 := @parameter0: java.lang.String[]"
    "$r1 = class \"Ljava/util/Locale;\""
    "$r0 = newarray (java.lang.Class)[3]"
    "$r0[0] = class \"Ljava/lang/String;\""
    "$r0[1] = class \"Ljava/lang/String;\""
    "$r0[2] = class \"Ljava/lang/String;\""
    "r37 = virtualinvoke $r1.<java.lang.Class: java.lang.reflect.Method getDeclaredMethod(java.lang.String,java.lang.Class[])>(\"getInstance\", $r0)"
    "virtualinvoke r37.<java.lang.reflect.Method: void setAccessible(boolean)>(1)"
    "$r2 = newarray (java.lang.Object)[3]"
    "$r2[0] = \"null\""
    "$r2[1] = \"GB\""
    "$r2[2] = \"\""
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r2)"
    "$r4 = newarray (java.lang.Object)[3]"
    "$r4[0] = null"
    "$r4[1] = \"GB\""
    "$r4[2] = \"\""
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r4)"
    "$r6 = new java.lang.RuntimeException"
    "specialinvoke $r6.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with language set to null\")"
    "throw $r6"
    "$r25 := @caughtexception"
    "r38 = $r25"
    "r39 = virtualinvoke r38.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"
    "$z2 = r39 instanceof java.lang.NullPointerException"
    "if $z2 != 0 goto $r26 = newarray (java.lang.Object)[3]"
    "$r31 = new java.lang.RuntimeException"
    "$r32 = new java.lang.StringBuilder"
    "specialinvoke $r32.<java.lang.StringBuilder: void <init>()>()"
    "$r33 = virtualinvoke $r32.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r39)"
    "$r34 = virtualinvoke $r33.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with language set to null\")"
    "$r35 = virtualinvoke $r34.<java.lang.StringBuilder: java.lang.String toString()>()"
    "specialinvoke $r31.<java.lang.RuntimeException: void <init>(java.lang.String)>($r35)"
    "throw $r31"
    "$r26 = newarray (java.lang.Object)[3]"
    "$r26[0] = \"en\""
    "$r26[1] = \"null\""
    "$r26[2] = \"\""
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r26)"
    "$r28 = newarray (java.lang.Object)[3]"
    "$r28[0] = \"en\""
    "$r28[1] = null"
    "$r28[2] = \"\""
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r28)"
    "$r30 = new java.lang.RuntimeException"
    "specialinvoke $r30.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with country set to null\")"
    "throw $r30"
    "$r14 := @caughtexception"
    "r40 = $r14"
    "r41 = virtualinvoke r40.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"
    "$z1 = r41 instanceof java.lang.NullPointerException"
    "if $z1 != 0 goto $r15 = newarray (java.lang.Object)[3]"
    "$r20 = new java.lang.RuntimeException"
    "$r21 = new java.lang.StringBuilder"
    "specialinvoke $r21.<java.lang.StringBuilder: void <init>()>()"
    "$r22 = virtualinvoke $r21.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r41)"
    "$r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with country set to null\")"
    "$r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.String toString()>()"
    "specialinvoke $r20.<java.lang.RuntimeException: void <init>(java.lang.String)>($r24)"
    "throw $r20"
    "$r15 = newarray (java.lang.Object)[3]"
    "$r15[0] = \"en\""
    "$r15[1] = \"GB\""
    "$r15[2] = \"null\""
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r15)"
    "$r17 = newarray (java.lang.Object)[3]"
    "$r17[0] = \"en\""
    "$r17[1] = \"GB\""
    "$r17[2] = null"
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r17)"
    "$r19 = new java.lang.RuntimeException"
    "specialinvoke $r19.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with variant set to null\")"
    "throw $r19"
    "$r8 := @caughtexception"
    "r42 = $r8"
    "r43 = virtualinvoke r42.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"
    "$z0 = r43 instanceof java.lang.NullPointerException"
    "if $z0 != 0 goto (branch)"
    "$r9 = new java.lang.RuntimeException"
    "$r10 = new java.lang.StringBuilder"
    "specialinvoke $r10.<java.lang.StringBuilder: void <init>()>()"
    "$r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r43)"
    "$r12 = virtualinvoke $r11.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with variant set to null\")"
    "$r13 = virtualinvoke $r12.<java.lang.StringBuilder: java.lang.String toString()>()"
    "specialinvoke $r9.<java.lang.RuntimeException: void <init>(java.lang.String)>($r13)"
    "throw $r9"
    "goto [?= return]"
    "$r7 := @caughtexception"
    "r44 = $r7"
    "return"
    "r36 := @parameter0: java.lang.String[]"->"$r1 = class \"Ljava/util/Locale;\"";
    "r36 := @parameter0: java.lang.String[]"->"$r7 := @caughtexception";
    "$r1 = class \"Ljava/util/Locale;\""->"$r0 = newarray (java.lang.Class)[3]";
    "$r1 = class \"Ljava/util/Locale;\""->"$r7 := @caughtexception";
    "$r0 = newarray (java.lang.Class)[3]"->"$r0[0] = class \"Ljava/lang/String;\"";
    "$r0 = newarray (java.lang.Class)[3]"->"$r7 := @caughtexception";
    "$r0[0] = class \"Ljava/lang/String;\""->"$r0[1] = class \"Ljava/lang/String;\"";
    "$r0[0] = class \"Ljava/lang/String;\""->"$r7 := @caughtexception";
    "$r0[1] = class \"Ljava/lang/String;\""->"$r0[2] = class \"Ljava/lang/String;\"";
    "$r0[1] = class \"Ljava/lang/String;\""->"$r7 := @caughtexception";
    "$r0[2] = class \"Ljava/lang/String;\""->"r37 = virtualinvoke $r1.<java.lang.Class: java.lang.reflect.Method getDeclaredMethod(java.lang.String,java.lang.Class[])>(\"getInstance\", $r0)";
    "$r0[2] = class \"Ljava/lang/String;\""->"$r7 := @caughtexception";
    "r37 = virtualinvoke $r1.<java.lang.Class: java.lang.reflect.Method getDeclaredMethod(java.lang.String,java.lang.Class[])>(\"getInstance\", $r0)"->"virtualinvoke r37.<java.lang.reflect.Method: void setAccessible(boolean)>(1)";
    "r37 = virtualinvoke $r1.<java.lang.Class: java.lang.reflect.Method getDeclaredMethod(java.lang.String,java.lang.Class[])>(\"getInstance\", $r0)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: void setAccessible(boolean)>(1)"->"$r2 = newarray (java.lang.Object)[3]";
    "virtualinvoke r37.<java.lang.reflect.Method: void setAccessible(boolean)>(1)"->"$r7 := @caughtexception";
    "$r2 = newarray (java.lang.Object)[3]"->"$r2[0] = \"null\"";
    "$r2 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r2[0] = \"null\""->"$r2[1] = \"GB\"";
    "$r2[0] = \"null\""->"$r7 := @caughtexception";
    "$r2[1] = \"GB\""->"$r2[2] = \"\"";
    "$r2[1] = \"GB\""->"$r7 := @caughtexception";
    "$r2[2] = \"\""->"virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r2)";
    "$r2[2] = \"\""->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r2)"->"$r25 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r2)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r2)"->"$r4 = newarray (java.lang.Object)[3]";
    "$r4 = newarray (java.lang.Object)[3]"->"$r25 := @caughtexception";
    "$r4 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r4 = newarray (java.lang.Object)[3]"->"$r4[0] = null";
    "$r4[0] = null"->"$r25 := @caughtexception";
    "$r4[0] = null"->"$r7 := @caughtexception";
    "$r4[0] = null"->"$r4[1] = \"GB\"";
    "$r4[1] = \"GB\""->"$r25 := @caughtexception";
    "$r4[1] = \"GB\""->"$r7 := @caughtexception";
    "$r4[1] = \"GB\""->"$r4[2] = \"\"";
    "$r4[2] = \"\""->"$r25 := @caughtexception";
    "$r4[2] = \"\""->"$r7 := @caughtexception";
    "$r4[2] = \"\""->"virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r4)";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r4)"->"$r25 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r4)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r4)"->"$r6 = new java.lang.RuntimeException";
    "$r6 = new java.lang.RuntimeException"->"$r25 := @caughtexception";
    "$r6 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "$r6 = new java.lang.RuntimeException"->"specialinvoke $r6.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with language set to null\")";
    "specialinvoke $r6.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with language set to null\")"->"$r25 := @caughtexception";
    "specialinvoke $r6.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with language set to null\")"->"$r7 := @caughtexception";
    "specialinvoke $r6.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with language set to null\")"->"throw $r6";
    "throw $r6"->"$r25 := @caughtexception";
    "throw $r6"->"$r7 := @caughtexception";
    "$r25 := @caughtexception"->"r38 = $r25";
    "$r25 := @caughtexception"->"$r7 := @caughtexception";
    "r38 = $r25"->"r39 = virtualinvoke r38.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()";
    "r38 = $r25"->"$r7 := @caughtexception";
    "r39 = virtualinvoke r38.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"->"$z2 = r39 instanceof java.lang.NullPointerException";
    "r39 = virtualinvoke r38.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"->"$r7 := @caughtexception";
    "$z2 = r39 instanceof java.lang.NullPointerException"->"if $z2 != 0 goto $r26 = newarray (java.lang.Object)[3]";
    "$z2 = r39 instanceof java.lang.NullPointerException"->"$r7 := @caughtexception";
    "if $z2 != 0 goto $r26 = newarray (java.lang.Object)[3]"->"$r31 = new java.lang.RuntimeException";
    "if $z2 != 0 goto $r26 = newarray (java.lang.Object)[3]"->"$r26 = newarray (java.lang.Object)[3]";
    "if $z2 != 0 goto $r26 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r31 = new java.lang.RuntimeException"->"$r32 = new java.lang.StringBuilder";
    "$r31 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "$r32 = new java.lang.StringBuilder"->"specialinvoke $r32.<java.lang.StringBuilder: void <init>()>()";
    "$r32 = new java.lang.StringBuilder"->"$r7 := @caughtexception";
    "specialinvoke $r32.<java.lang.StringBuilder: void <init>()>()"->"$r33 = virtualinvoke $r32.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r39)";
    "specialinvoke $r32.<java.lang.StringBuilder: void <init>()>()"->"$r7 := @caughtexception";
    "$r33 = virtualinvoke $r32.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r39)"->"$r34 = virtualinvoke $r33.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with language set to null\")";
    "$r33 = virtualinvoke $r32.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r39)"->"$r7 := @caughtexception";
    "$r34 = virtualinvoke $r33.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with language set to null\")"->"$r35 = virtualinvoke $r34.<java.lang.StringBuilder: java.lang.String toString()>()";
    "$r34 = virtualinvoke $r33.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with language set to null\")"->"$r7 := @caughtexception";
    "$r35 = virtualinvoke $r34.<java.lang.StringBuilder: java.lang.String toString()>()"->"specialinvoke $r31.<java.lang.RuntimeException: void <init>(java.lang.String)>($r35)";
    "$r35 = virtualinvoke $r34.<java.lang.StringBuilder: java.lang.String toString()>()"->"$r7 := @caughtexception";
    "specialinvoke $r31.<java.lang.RuntimeException: void <init>(java.lang.String)>($r35)"->"throw $r31";
    "specialinvoke $r31.<java.lang.RuntimeException: void <init>(java.lang.String)>($r35)"->"$r7 := @caughtexception";
    "throw $r31"->"$r7 := @caughtexception";
    "$r26 = newarray (java.lang.Object)[3]"->"$r26[0] = \"en\"";
    "$r26 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r26[0] = \"en\""->"$r26[1] = \"null\"";
    "$r26[0] = \"en\""->"$r7 := @caughtexception";
    "$r26[1] = \"null\""->"$r26[2] = \"\"";
    "$r26[1] = \"null\""->"$r7 := @caughtexception";
    "$r26[2] = \"\""->"virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r26)";
    "$r26[2] = \"\""->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r26)"->"$r14 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r26)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r26)"->"$r28 = newarray (java.lang.Object)[3]";
    "$r28 = newarray (java.lang.Object)[3]"->"$r14 := @caughtexception";
    "$r28 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r28 = newarray (java.lang.Object)[3]"->"$r28[0] = \"en\"";
    "$r28[0] = \"en\""->"$r14 := @caughtexception";
    "$r28[0] = \"en\""->"$r7 := @caughtexception";
    "$r28[0] = \"en\""->"$r28[1] = null";
    "$r28[1] = null"->"$r14 := @caughtexception";
    "$r28[1] = null"->"$r7 := @caughtexception";
    "$r28[1] = null"->"$r28[2] = \"\"";
    "$r28[2] = \"\""->"$r14 := @caughtexception";
    "$r28[2] = \"\""->"$r7 := @caughtexception";
    "$r28[2] = \"\""->"virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r28)";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r28)"->"$r14 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r28)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r28)"->"$r30 = new java.lang.RuntimeException";
    "$r30 = new java.lang.RuntimeException"->"$r14 := @caughtexception";
    "$r30 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "$r30 = new java.lang.RuntimeException"->"specialinvoke $r30.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with country set to null\")";
    "specialinvoke $r30.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with country set to null\")"->"$r14 := @caughtexception";
    "specialinvoke $r30.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with country set to null\")"->"$r7 := @caughtexception";
    "specialinvoke $r30.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with country set to null\")"->"throw $r30";
    "throw $r30"->"$r14 := @caughtexception";
    "throw $r30"->"$r7 := @caughtexception";
    "$r14 := @caughtexception"->"r40 = $r14";
    "$r14 := @caughtexception"->"$r7 := @caughtexception";
    "r40 = $r14"->"r41 = virtualinvoke r40.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()";
    "r40 = $r14"->"$r7 := @caughtexception";
    "r41 = virtualinvoke r40.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"->"$z1 = r41 instanceof java.lang.NullPointerException";
    "r41 = virtualinvoke r40.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"->"$r7 := @caughtexception";
    "$z1 = r41 instanceof java.lang.NullPointerException"->"if $z1 != 0 goto $r15 = newarray (java.lang.Object)[3]";
    "$z1 = r41 instanceof java.lang.NullPointerException"->"$r7 := @caughtexception";
    "if $z1 != 0 goto $r15 = newarray (java.lang.Object)[3]"->"$r20 = new java.lang.RuntimeException";
    "if $z1 != 0 goto $r15 = newarray (java.lang.Object)[3]"->"$r15 = newarray (java.lang.Object)[3]";
    "if $z1 != 0 goto $r15 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r20 = new java.lang.RuntimeException"->"$r21 = new java.lang.StringBuilder";
    "$r20 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "$r21 = new java.lang.StringBuilder"->"specialinvoke $r21.<java.lang.StringBuilder: void <init>()>()";
    "$r21 = new java.lang.StringBuilder"->"$r7 := @caughtexception";
    "specialinvoke $r21.<java.lang.StringBuilder: void <init>()>()"->"$r22 = virtualinvoke $r21.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r41)";
    "specialinvoke $r21.<java.lang.StringBuilder: void <init>()>()"->"$r7 := @caughtexception";
    "$r22 = virtualinvoke $r21.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r41)"->"$r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with country set to null\")";
    "$r22 = virtualinvoke $r21.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r41)"->"$r7 := @caughtexception";
    "$r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with country set to null\")"->"$r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.String toString()>()";
    "$r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with country set to null\")"->"$r7 := @caughtexception";
    "$r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.String toString()>()"->"specialinvoke $r20.<java.lang.RuntimeException: void <init>(java.lang.String)>($r24)";
    "$r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.String toString()>()"->"$r7 := @caughtexception";
    "specialinvoke $r20.<java.lang.RuntimeException: void <init>(java.lang.String)>($r24)"->"throw $r20";
    "specialinvoke $r20.<java.lang.RuntimeException: void <init>(java.lang.String)>($r24)"->"$r7 := @caughtexception";
    "throw $r20"->"$r7 := @caughtexception";
    "$r15 = newarray (java.lang.Object)[3]"->"$r15[0] = \"en\"";
    "$r15 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r15[0] = \"en\""->"$r15[1] = \"GB\"";
    "$r15[0] = \"en\""->"$r7 := @caughtexception";
    "$r15[1] = \"GB\""->"$r15[2] = \"null\"";
    "$r15[1] = \"GB\""->"$r7 := @caughtexception";
    "$r15[2] = \"null\""->"virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r15)";
    "$r15[2] = \"null\""->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r15)"->"$r8 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r15)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r15)"->"$r17 = newarray (java.lang.Object)[3]";
    "$r17 = newarray (java.lang.Object)[3]"->"$r8 := @caughtexception";
    "$r17 = newarray (java.lang.Object)[3]"->"$r7 := @caughtexception";
    "$r17 = newarray (java.lang.Object)[3]"->"$r17[0] = \"en\"";
    "$r17[0] = \"en\""->"$r8 := @caughtexception";
    "$r17[0] = \"en\""->"$r7 := @caughtexception";
    "$r17[0] = \"en\""->"$r17[1] = \"GB\"";
    "$r17[1] = \"GB\""->"$r8 := @caughtexception";
    "$r17[1] = \"GB\""->"$r7 := @caughtexception";
    "$r17[1] = \"GB\""->"$r17[2] = null";
    "$r17[2] = null"->"$r8 := @caughtexception";
    "$r17[2] = null"->"$r7 := @caughtexception";
    "$r17[2] = null"->"virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r17)";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r17)"->"$r8 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r17)"->"$r7 := @caughtexception";
    "virtualinvoke r37.<java.lang.reflect.Method: java.lang.Object invoke(java.lang.Object,java.lang.Object[])>(null, $r17)"->"$r19 = new java.lang.RuntimeException";
    "$r19 = new java.lang.RuntimeException"->"$r8 := @caughtexception";
    "$r19 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "$r19 = new java.lang.RuntimeException"->"specialinvoke $r19.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with variant set to null\")";
    "specialinvoke $r19.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with variant set to null\")"->"$r8 := @caughtexception";
    "specialinvoke $r19.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with variant set to null\")"->"$r7 := @caughtexception";
    "specialinvoke $r19.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Should NPE with variant set to null\")"->"throw $r19";
    "throw $r19"->"$r8 := @caughtexception";
    "throw $r19"->"$r7 := @caughtexception";
    "$r8 := @caughtexception"->"r42 = $r8";
    "$r8 := @caughtexception"->"$r7 := @caughtexception";
    "r42 = $r8"->"r43 = virtualinvoke r42.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()";
    "r42 = $r8"->"$r7 := @caughtexception";
    "r43 = virtualinvoke r42.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"->"$z0 = r43 instanceof java.lang.NullPointerException";
    "r43 = virtualinvoke r42.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getCause()>()"->"$r7 := @caughtexception";
    "$z0 = r43 instanceof java.lang.NullPointerException"->"if $z0 != 0 goto (branch)";
    "$z0 = r43 instanceof java.lang.NullPointerException"->"$r7 := @caughtexception";
    "if $z0 != 0 goto (branch)"->"$r9 = new java.lang.RuntimeException";
    "if $z0 != 0 goto (branch)"->"goto [?= return]";
    "if $z0 != 0 goto (branch)"->"$r7 := @caughtexception";
    "$r9 = new java.lang.RuntimeException"->"$r10 = new java.lang.StringBuilder";
    "$r9 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "$r10 = new java.lang.StringBuilder"->"specialinvoke $r10.<java.lang.StringBuilder: void <init>()>()";
    "$r10 = new java.lang.StringBuilder"->"$r7 := @caughtexception";
    "specialinvoke $r10.<java.lang.StringBuilder: void <init>()>()"->"$r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r43)";
    "specialinvoke $r10.<java.lang.StringBuilder: void <init>()>()"->"$r7 := @caughtexception";
    "$r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r43)"->"$r12 = virtualinvoke $r11.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with variant set to null\")";
    "$r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.Object)>(r43)"->"$r7 := @caughtexception";
    "$r12 = virtualinvoke $r11.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with variant set to null\")"->"$r13 = virtualinvoke $r12.<java.lang.StringBuilder: java.lang.String toString()>()";
    "$r12 = virtualinvoke $r11.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(\" is thrown with variant set to null\")"->"$r7 := @caughtexception";
    "$r13 = virtualinvoke $r12.<java.lang.StringBuilder: java.lang.String toString()>()"->"specialinvoke $r9.<java.lang.RuntimeException: void <init>(java.lang.String)>($r13)";
    "$r13 = virtualinvoke $r12.<java.lang.StringBuilder: java.lang.String toString()>()"->"$r7 := @caughtexception";
    "specialinvoke $r9.<java.lang.RuntimeException: void <init>(java.lang.String)>($r13)"->"throw $r9";
    "specialinvoke $r9.<java.lang.RuntimeException: void <init>(java.lang.String)>($r13)"->"$r7 := @caughtexception";
    "throw $r9"->"$r7 := @caughtexception";
    "goto [?= return]"->"return";
    "$r7 := @caughtexception"->"r44 = $r7";
    "r44 = $r7"->"return";
}
